src/mon/Monitor.cc

```diff
@@ -3271,6 +3271,11 @@ void Monitor::handle_route(MonOpRequestRef op)
 	rr->con->send_message(m->msg);
 	m->msg = NULL;
       }
+      if (m->send_osdmap_first) {
+	dout(10) << " sending osdmaps from " << m->send_osdmap_first << dendl;
+	osdmon()->send_incremental(m->send_osdmap_first, session,
+				   true, MonOpRequestRef());
+      }
```

Comment:
I don't think it's required, but the name "send_osdmap_first" definitely makes me think we're sending the OSDMaps back before the normal response. And given the code structure I think that's the more likely pattern if we add anything that does require a separate response besides the maps.